Formula/ntfy.rb

```diff
@@ -0,0 +1,42 @@
+class Ntfy < Formula
+  desc "Send push notifications to your phone or desktop via PUT/POST"
+  homepage "https://ntfy.sh/"
+  url "https://github.com/binwiederhier/ntfy.git", tag: "v2.3.1", revision: "a75fb08ef138d67599499c1f78628a0b35fcef54"
```

Comment:
```suggestion
  url "https://github.com/binwiederhier/ntfy.git",
      tag: "v2.3.1",
      revision: "a75fb08ef138d67599499c1f78628a0b35fcef54"
```
Let's follow the format used by other formulae. Mind the spacing for `tag:` and `revision:`. They are aligned with the first argument to `url`.

Reply:
Sure! Thanks for pointing this out. Maybe this can also be added as the check, in the brew style check?

Reply:
Yes, ideally that would be, but we haven't gotten around to adding it to our style checks.